browser/components/loop/content/css/contacts.css

```diff
@@ -7,6 +7,67 @@ html {
   font-family: sans-serif; /* XXX will be changed to a system font in bug 1191398 */
 }
 
+.contacts-container {
+  flex: 1;
+  display: flex;
+  flex-flow: column nowrap;
+  overflow: auto;
+}
+
+.contact-list-container {
+  flex: 1;
+  display: flex;
+  flex-direction: column;
+  overflow: auto;
+}
+/* Don't show the Gravatar if we're showing a contacts list. */
+.contact-list ~ .contacts-gravatar-promo {
+  display: none;
+}
+
+.contact-list-wrapper {
+  flex: 2;
+  overflow-x: hidden;
+  overflow-y: auto;
+}
+
+/* Don't show the Gravatar if we're showing a contacts list. NOT WORKING hide in code*/
+.contact-search-list-empty ~ .contacts-gravatar-promo {
+  display: block;
+}
+
+/* Don't show the empty contacts image if we're showing gravatar promo. */
+.contacts-gravatar-promo ~ .contact-list-empty {
+  background-image: none;
+  padding-top: 0;
+  padding-bottom: 0;
+  margin-top: 2px;
+}
+
+.contact-list-empty-container {
+  flex: 1;
+  overflow-x: hidden;
+  overflow-y: auto;
+}
+
+.contact-list-empty,
+.contact-search-list-empty {
+  margin-top: 4rem;
+  padding-top: 11.5rem;
+  padding-bottom: 0;
+  text-align: center;
+  color: #4a4a4a;
+}
```

Comment:
Removed duplicate css from contacts and left panel.css shared css properties. Done